### PR TITLE
Reader: Update Following and Manage empty state

### PR DESCRIPTION
**Pull request by jancavan**
Following (Before):
screenshot 2017-10-13 13 26 18

Following (After):
screenshot 2017-10-13 13 29 54

For new accounts, we auto-follow them to the Big 5 so the empty stream will only be seen if a user decides to unfollow these 5 sites.

Manage (Before):
screenshot 2017-10-13 13 26 08

Manage (After):
screenshot 2017-10-13 13 29 58

### CODE REVIEW

**Comment by fraying** [comment]
So great! Yay @jancavan !

File: client/reader/following-manage/empty.jsx [hunk 1]
```diff
@@ -36,12 +36,12 @@ class FollowingManageEmptyContent extends React.Component {
 
 		return (
 			<EmptyContent
-				className="is-drake"
+				className="is-following-manage-empty"
```
Comment (bluefuton) [inline client/reader/following-manage/empty.jsx]: I originally used `is-drake` as a modifier to apply different spacing all of the empty pages that used Drake.

I think we can just drop the `is-` prefix now and use `following-manage__empty` here.

File: client/reader/stream/empty.jsx [hunk 2]
```diff
@@ -37,13 +37,13 @@ class FollowingEmptyContent extends React.Component {
 		/* eslint-disable wpcalypso/jsx-classname-namespace */
 		return (
 			<EmptyContent
-				className="is-drake"
+				className="is-following-empty"
```
Comment (bluefuton) [inline client/reader/stream/empty.jsx]: I think we can just drop the `is-` prefix now and use `stream__empty` here to match the component.

**Comment by bluefuton** [comment]
Class naming aside, I tested both empty states and they look great!

**Comment by jancavan** [comment]
@bluefuton I renamed the classes in: https://github.com/Automattic/wp-calypso/pull/18835/commits/80fa603f1f4c5d543ad111801c4597fc337795b6, but looking at the rendered html, it does follow the component's main class `empty-content`. So I think it makes sense for it a modifier?

**Pull request merged**